CompoundV3-Bots/Reserves-Monitor/src/agent.spec.ts

```diff
@@ -0,0 +1,337 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleBlock,
+  BlockEvent,
+} from "forta-agent";
+import { RESERVES_ABI, TARGET_RESERVES_ABI } from "./constants";
+
+import {
+  createAddress,
+  TestBlockEvent,
+  MockEthersProvider,
+} from "forta-agent-tools/lib/tests";
+import { BigNumber } from "ethers";
+import { Interface } from "ethers/lib/utils";
+import Fetcher from "./dataFetcher";
+
+const TEST_ADDRESSES = [createAddress("0x11"), createAddress("0x12")];
+const mockProvider: MockEthersProvider = new MockEthersProvider();
+const ALERT_FREQ = 1000;
+const mockGetFn = (id: string) => {
+  if (id == "cometAddresses") return TEST_ADDRESSES;
+  else return ALERT_FREQ;
+};
+const mockNetworkManager = {
+  cometAddresses: TEST_ADDRESSES,
+  networkMap: {},
+  setNetwork: jest.fn(),
+  get: mockGetFn as any,
+};
+
+const IFACE = new Interface([RESERVES_ABI, TARGET_RESERVES_ABI]);
+const INIT_TIMESTAMP = 10000;
+
+describe("COMP2 - Reserves Monitor Bot Tests suite", () => {
```

Comment:
Good, I'll try to keep a standard in my test suite titles too. But just for the sake of consistency:
```suggestion
describe("COMP2-1 - Reserves Monitor Bot Test Suite", () => {
```